cms/templates/home_page.html

```diff
@@ -17,13 +17,20 @@
   {{ block.super }}
   <script type="text/javascript" src="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
   <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/hls.js@latest"></script>
+  <script src="{% static 'js/jquery.fancybox.min.js' %}"></script>
```

Comment:
Can you move this to `js/jquery.fancybox.min.js` to `js/vendor/jquery.fancybox.min.js`, and `css/jquery.fancybox.min.css` to `css/vendor/jquery.fancybox.min.css`?